eng/pipelines/common/xplat-setup.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       runtimeFlavorDisplayName: 'CoreCLR'
     ${{ if eq(parameters.jobParameters.runtimeFlavor, 'mono') }}:
       runtimeFlavorDisplayName: 'Mono'
+    
+    shouldContinueOnError: ${{ endsWith(variables['Build.DefinitionName'], 'staging') }}
+
```

Comment:
xplat-setup.yml is a common template for all tests? Since I see the check for staging added only here, I want to make sure this template is used by all paths possible.

Reply:
Yeah, this template is included in all paths on our pipelines. Basically all jobs are template references to `platform-matrix.yml` which then expands `xplat-setup.yml` for each job.